lib/titles.js

```diff
@@ -131,4 +131,5 @@ export {
   getNamespaceByTitle,
   productMapping,
   productConfig,
+  validNamespaces,
```

Comment:
Shouldn't this be in the namespaces file instead?

Reply:
nej